charts/internal/calico/templates/node/daemonset-calico-node.yaml

```diff
@@ -61,6 +61,34 @@ spec:
         securityContext:
           privileged: true
       {{- end }}
+      # Log MTU issues if monitoring is enabled
+      {{- if .Values.config.monitoring.enabled }}
+      - name: log-mtu-issues
+        image: {{ index .Values.images "calico-node" }}
+        command:
+          - /bin/sh
+          - -c
+          - |
+            if [ -x "/usr/sbin/iptables-legacy" ]; then
+              backend="legacy"
+            else
+              backend="nft"
+            fi
```

Comment:
For this rules it might not  be important. But for consistency we should decide between legacy and nft based on the kublet rules like  [here](https://github.com/gardener/gardener-extension-networking-calico/blob/7cc6703866cba7f8892b8df22a97a969ca05a8ab/charts/internal/calico/templates/node/daemonset-calico-node.yaml#L180-L185)

Reply:
thanks, I've tested it and it works fine. updated the PR accordingly.